src/Parser/RestoDruid/Modules/Features/Mastery.js

```diff
@@ -0,0 +1,143 @@
+import {formatPercentage} from 'common/format';
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import {ABILITIES_AFFECTED_BY_HEALING_INCREASES} from '../../Constants';
+import {HEALS_MASTERY_STACK} from '../../Constants';
+
+const MASTERY_BONUS_FROM_ONE_RATING = 1 / 66666.6666666;
+const BASE_MASTERY_PERCENT = 0.048;
+
+class Mastery extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  totalNoMasteryHealing = 0;
+  druidSpellNoMasteryHealing = 0;
+  masteryTimesHealing = 0;
+
+  hotHealingMap = null;
+  masteryBuffs = null;
+
+  on_initialized() {
+    // TODO use JS objects instead of Maps for hotHealingMap and masteryBuffs?
+    this.hotHealingMap = new Map();
+    for(const healId of HEALS_MASTERY_STACK) {
+      this.hotHealingMap.set(healId, {'name':SPELLS[healId].name, 'direct':0, 'mastery':0});
```

Comment:
Name seems unused and can probably be omitted. In addition if you want to show a list, I'd prefer only fetching the name once you want to render it rather than in a memory object (using the entire spell object is also good).

Also the property names of objects do not have to be quoted.

Reply:
I think I included more than I needed when designing this because I hadn't yet nailed down the design. I'll get to fixing this in next PR.